### DISCUSSION TITLE
Annotations - Close popup with "x" button in template?

### DISCUSSION THREAD

**Discussion by camya** [xeokit/xeokit-sdk#1589] [Q&A]
I created annotation markers using the [createAnnotation](https://xeokit.github.io/xeokit-sdk/docs/class/src/plugins/AnnotationsPlugin/AnnotationsPlugin.js~AnnotationsPlugin.html#instance-method-createAnnotation) method.

- Each marker popup should contain a **close button**.  (See **[x]** in screenshot)
- If the user clicks the x button, it closes the popup.

How can I archive this with the xeokit-sdk?

![image](https://github.com/user-attachments/assets/5bbab1b5-b9af-4796-bf15-579dcef71792)

**Comment by xeolabs** [comment]
See this example here, where we toggle the display of the labels using `Annotation.setLabelShown()`: 

https://xeokit.github.io/xeokit-sdk/examples/annotations/#annotations_clickShowLabels

https://github.com/xeokit/xeokit-sdk/blob/master/examples/annotations/annotations_clickShowLabels.html#L174

You could hook that up to the close button.
